src/libraries/System.Private.CoreLib/src/System/Array.cs

```diff
@@ -1460,20 +1460,20 @@ public static void Reverse(Array array, int index, int length)
                     return;
                 case CorElementType.ELEMENT_TYPE_I4:
                 case CorElementType.ELEMENT_TYPE_U4:
+                case CorElementType.ELEMENT_TYPE_R4:
```

Comment:
It would be nice to minimize this diff by putting `R4` back where it was originally (same with `R8` below).